build/quicktype-core/package.in.json

```diff
@@ -20,15 +20,13 @@
         "pluralize": "^7.0.0",
         "@mark.probst/unicode-properties": "~1.1.0",
         "urijs": "^1.19.1",
-        "wordwrap": "^1.0.0",
-        "yaml": "^1.5.0"
+        "wordwrap": "^1.0.0"
```

Comment:
Why is YAML removed here?

Reply:
It must have been changed and mistakenly committed when I was getting my build and test environment up and running. I've put it back.